storybook/pages/CommunitiesPortalLayoutPage.qml

```diff
@@ -1,57 +1,105 @@
+import Storybook 1.0
+
+import utils 1.0
+
+SplitView {
+    QtObject {
+        id: internal
+
+        readonly property var logEvent: logsAndControlsPanel.logsView.logEvent
+    }
+
+    SplitView {
+        orientation: Qt.Vertical
+        SplitView.fillWidth: true
 
-    CommunitiesPortalLayout {
-        Layout.fillWidth: true
-        Layout.fillHeight: true
+        ColumnLayout {
+            SplitView.fillWidth: true
+            SplitView.fillHeight: true
 
-        communitiesStore: CommunitiesStore {
-            readonly property string locale: ""
-            readonly property int unreadNotificationsCount: 42
-            readonly property string communityTags:
-                JSON.stringify({"Activism":"✊","Art":"🎨","Blockchain":"🔗","Books & blogs":"📚","Career":"💼"})
-            readonly property var curatedCommunitiesModel: SortFilterProxyModel {
+            CommunitiesPortalLayout {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
 
-                sourceModel: CommunitiesPortalDummyModel {}
+                communitiesStore: CommunitiesStore {
+                    readonly property string locale: ""
+                    readonly property int unreadNotificationsCount: 42
+                    readonly property string communityTags:
+                        JSON.stringify({"Activism":"✊","Art":"🎨","Blockchain":"🔗","Books & blogs":"📚","Career":"💼"})
+                    readonly property var curatedCommunitiesModel: SortFilterProxyModel {
+
+                        sourceModel: CommunitiesPortalDummyModel { id: mockedModel }
+
+                        filters: IndexFilter {
+                            inverted: true
+                            minimumIndex: Math.floor(slider.value)
+                        }
+                    }
+
+                    function setActiveCommunity() {
+                        internal.logEvent("CommunitiesStore::setActiveCommunity", ["communityId"], arguments)
+                    }
+                }
+
+                // TODO: onCompleted handler and localAccountSensitiveSettings are here to allow opening
+                // "Import Community" and "Create New Community" popups. However those popups shouldn't
+                // be tightly coupled with `CommunitiesPortalLayout` so it should be refactored in the next step.
+                // Pressing buttons "Import using key" and "Create new community" should only request for opening
+                // dialogs, and in Storybook it should be logged in the same way as calls to stores.
+                // Mentioned popups should have their own pages in the Storybook.
+                Component.onCompleted: {
+                    Global.appMain = this
+                }
 
-                filters: IndexFilter {
-                    inverted: true
-                    minimumIndex: Math.floor(slider.value)
+                QtObject {
+                    id: localAccountSensitiveSettings
```

Comment:
This is coming from NIM... no idea how to mock that properly or more efficiently